.github/workflows/build-git-installers.yml

```diff
@@ -187,24 +187,17 @@ jobs:
         run: |
           set -x
 
-          eval /usr/src/build-extra/please.sh make_installers_from_mingw_w64_git --version=${{ needs.prereqs.outputs.tag_version }} -o artifacts --${{matrix.artifact.name}} --pkg=pkg-x86_64/mingw-w64-x86_64-git-[0-9]*.tar.xz --pkg=pkg-x86_64/mingw-w64-x86_64-git-doc-html-[0-9]*.tar.xz &&
+          b="/usr/src/build-extra" &&
+          mkdir -p $b/cached-source-packages &&
+          cp pkg-x86_64/*-pdb* $b/cached-source-packages/ &&
```

Comment:
Does this line need to be after the `eval`?

Reply:
The PDB archive needs to be "staged" in the `cached-source-packages` directory before `make_installers_from_mingw_w64_git` is run with the `--include-pdbs` flag (this pull request added that option: https://github.com/git-for-windows/build-extra/pull/369).